airqo-platform/AirQo-frontend
# [NetManager] Add other pollutants

#### Summary of Changes (What does this PR do?)
- Adds access to other pollutants on NetManager map.

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch this branch locally
* Install requirements using `npm install`
* Start Application using `npm run stage-mac` (MacOs) or `npm run stage-pc` (Windows PC)
* Should be able to access all the other pollutants on netmanager map.

#### What are the relevant tickets?
- [PLAT-580](https://airqoteam.atlassian.net/browse/PLAT-580)

#### Screenshots (optional)
